### PR TITLE
Test with MySQL/MariaDB ONLY_FULL_GROUP_BY

### PR DESCRIPTION
**Pull request by nickvergessen**
Ref https://github.com/nextcloud/spreed/pull/7031/commits/9d5cfbdd8dc8b0f27fb2357c9be62a6dd692f23b
Ref https://github.com/nextcloud/documentation/pull/8139

### CODE REVIEW

File: .github/workflows/phpunit-mysql.yml [hunk 1]
```diff
@@ -35,6 +35,11 @@ jobs:
         options: --health-cmd="mysqladmin ping" --health-interval 5s --health-timeout 2s --health-retries 5
 
     steps:
+      - name: Enable ONLY_FULL_GROUP_BY MySQL option
+        run: |
+          echo "SET GLOBAL sql_mode=(SELECT CONCAT(@@sql_mode,',ONLY_FULL_GROUP_BY'));" | mysql -h 127.0.0.1 -P 4444 -u root -prootpassword
```
Comment (skjnldsv) [inline .github/workflows/phpunit-mysql.yml:40]: Can we not use https://mariadb.com/kb/en/server-system-variables/#sql_mode instead?
In the service start parameters?

Comment (nickvergessen) [reply .github/workflows/phpunit-mysql.yml:40]: I use that in drone where I found out how to apply it:
https://github.com/nextcloud/spreed/pull/7032/files#diff-5b4c4090a99df0c5d2b12cf660513eb64bbd1b209c208c938bc6e0bef2400e7bR81

But I couldn't find the thing for github actions to pipe it down to the mysql running inside a docker.

Comment (nickvergessen) [reply .github/workflows/phpunit-mysql.yml]: Pushed a patch, let's see

File: .github/workflows/phpunit-mysql.yml [hunk 2]
```diff
@@ -33,8 +33,13 @@ jobs:
         env:
           MYSQL_ROOT_PASSWORD: rootpassword
         options: --health-cmd="mysqladmin ping" --health-interval 5s --health-timeout 2s --health-retries 5
+        commands:
+          - --sql-mode=ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_ENGINE_SUBSTITUTION
```
Comment (nickvergessen) [inline .github/workflows/phpunit-mysql.yml:37]: The workflow is not valid. .github/workflows/phpunit-mysql.yml (Line: 36, Col: 9): Unexpected value 'commands'

File: .github/workflows/phpunit-mysql.yml [hunk 3]
```diff
@@ -32,9 +32,12 @@ jobs:
           - 4444:3306/tcp
         env:
           MYSQL_ROOT_PASSWORD: rootpassword
-        options: --health-cmd="mysqladmin ping" --health-interval 5s --health-timeout 2s --health-retries 5
+        options: --health-cmd="mysqladmin ping" --health-interval 5s --health-timeout 2s --health-retries 5 --sql-mode=ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_ENGINE_SUBSTITUTION
```
[the remark this answers is not in the document]
Comment (skjnldsv) [reply .github/workflows/phpunit-mysql.yml:35]: Actually doesn't seems to be supported.
Let's go with your hack

**Review by skjnldsv — APPROVED** [review]

**Pull request merged**